app/scripts/controllers/StudentsIndexCtrl.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+define(['paw'], function(paw) {
+  paw.controller('StudentsIndexCtrl', function() {
+    var _this = this;
+    this.docket = '';
```

Comment:
I would not assign all these variables direct to the controller, but instead to a filter's object.
```js
this.filter = {
  docket: '',
  firstName: '',
  lastName: ''
}
```